### DISCUSSION TITLE
Go Build using Github Actions `FUNCTION_INVOCATION_FAILED`

### DISCUSSION THREAD

**Discussion by afifurrohman-id** [vercel/community#3777] [Help]
### Page to Investigate

https://github.com/afifurrohman-id/go-hello-api/actions/runs/5982730800

### Steps to Reproduce

- Ci Cd Success: https://github.com/afifurrohman-id/go-hello-api/actions/runs/5982730800
- But App is not run: https://go-hello-okkw8n9tf-afif-id.vercel.app/

**No Error log on vercel deployment**

**Comment by amyegan** [comment]
Hey @afifurrohman-id. It looks like you already solved this. Are you still having trouble with the app?
